src/app/learn/reference/component-libraries/index.html

```diff
@@ -177,4 +177,24 @@
     }
   </sky-code-block>
 
+  <stache-page-anchor>
+    Test modules
+  </stache-page-anchor>
+
+  <p>
+    You can generate a separate module for testing by adding a test directory. The test module uses a separate test file to prevent cluttering the production module during webpack builds.
+  </p>
+
+  <sky-code-block>
```

Comment:
This code block seems unnecessary. It seems like this should just be a code snippet combined with the next paragraph. What do you think of:
"To create a testing module, add a `testing` directory to `src/app/public` and then add an `index.ts` file. The library exports the contents of the `index.ts` file to NPM as a testing module."